p/scripts/main.js

```diff
@@ -803,7 +803,9 @@ function init_shortcuts() {
 				if (context.auto_mark_site) {
 					mark_read(document.querySelector('.flux.current'), true, false);
 				}
-				window.open(document.querySelector('.flux.current a.go_website').href);
+				const newWindow = window.open();
```

Comment:
I assume you mean this?

```suggestion
				const newWindow = window.open;
```

Reply:
Yes, the SO thread I linked seemed hesitant about browser compatibility, and according to your README IE11 is supported, so I went with the old fashion method. Let me know if you prefer the other one, I'll update the PR.

Reply:
For reference https://developer.mozilla.org/docs/Web/API/Window/open